functorch/_src/make_functional.py

```diff
@@ -353,15 +359,21 @@ def compute_loss(params, x, t):
 
     If the model has any buffers, please use :func:`make_functional_with_buffers` instead.
 
+    Args:
+        model (torch.nn.Module): Input model.
+        params_require_grad (bool): If True, output parameters will have require gradient.
+            It could be helpful to set this flag to False if computions do not need higher order gradients.
```

Comment:
nit: maybe "computations do not need higher order gradients" -> "gradient computations only use the functorch API"

This point definitely got jumbled in the discussion on the issue, but basically we can still get higher order gradients with this, we just would have to use functorch's `grad(grad(...))`. The issue will be if we did something like `grad(...).backward()`

Reply:
@samdow could you please check again the docstring and maybe help to improve it.